src/components/tree-view/tree-view.css

```diff
@@ -0,0 +1,4 @@
+[data-selected="true"] {
+    background-color: red;
```

Comment:
Please, this is default. It burned both of my retinas. Also, add indention to the children. Pretty please.